tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2222,6 +2223,37 @@ describe('scalar card', () => {
           }),
         ]);
       }));
+
+      it('toggles selected time when single fob is deselected', fakeAsync(() => {
```

Comment:
Should we also add a test for range selection, which emits `timeSelectionChanged`?